recipes/srt/all/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(CONAN_LIBS_PTHREADS4W)
+    set(PTHREAD_LIBRARY "${CONAN_LIBS_PTHREADS4W}")
+    set(PTHREAD_INCLUDE_DIR "${CONAN_INCLUDE_DIRS_PTHREADS4W}")
+endif()
```

Comment:
Is this a common pattern in other recipes? Is it possible to `find_package(pthreads4w)` and use the provided targets?

Reply:
`pthreads4w` does not have CMake support so there's no expectation from consumer to find it. This project provides options for providing them... they could be moved to the recipe under `_configure_cmake()` however I do not know how to extract these settings from cpp_info.